pkg/arvo/sys/zuse.hoon

```diff
@@ -2155,6 +2155,114 @@
         [r s y.rp]
       ::  general recovery omitted, but possible
       --
+    ::
+    ++  pedersen
+      ~%  %pedersen  +  ~
```

Comment:
I also am not seeing a ~/  %phash  hint anywhere... does this jet currently fire?